src/generator.py

```
@@ -0,0 +1,29 @@
+import os
+from MDtoHTML import markdown_to_html_node
+def extract_title(markdown: str):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)